Code/GraphMol/Chirality.cpp

```diff
@@ -402,9 +402,17 @@ void findAtomNeighborsHelper(const ROMol &mol, const Atom *atom,
   }
 }
 
+// conditions for an atom to be a candidate for ring stereochem:
+//   1) two non-ring neighbors that have different ranks
+//   2) one non-ring neighbor and two ring neighbors (the ring neighbors will
+//      have the same rank)
+//   3) four ring neighbors with three different ranks
```

Comment:
Hmm, no. If it had four different ranks it would be chiral and we wouldn't be here. The code does say >=3, but it really could be ==3.